src/test/java/theinternet/forms/WelcomeForm.java

```diff
@@ -0,0 +1,15 @@
+package theinternet.forms;
+
+import org.openqa.selenium.By;
+
+public class WelcomeForm extends TheInternetForm {
+
+    public WelcomeForm() {
+        super(By.xpath("//h1[contains(.,'Welcome to the-internet')]"), "Welcome to the-internet");
+    }
+
+    @Override
+    protected String getUri() {
+        return "";
```

Comment:
is that correct that it is empty??

Reply:
yes. as a result we will have start url for site http://the-internet.herokuapp.com